redaxo/src/core/lib/console/user/create.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @internal
+ */
+class rex_command_user_create extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Create a new user')
+            ->addArgument('login', InputArgument::REQUIRED, 'Login')
+            ->addArgument('password', InputArgument::OPTIONAL, 'Password')
+            ->addOption('username', null, InputOption::VALUE_OPTIONAL, 'Username')
```

Comment:
```suggestion
            ->addOption('username', null, InputOption::VALUE_REQUIRED, 'Username')
```

Optionen sind immer optional. Wenn man diese Option verwendet, dann ist ein Value aber erforderlich.

Reply:
passt dieser vorschlag dazu, wenn man nur `--admin` notieren will?

Reply:
oder müsste es dann VALUE_NONE sein ,wenn ich es machen will wie von dir weiter oben vorgeschlagen?

Reply:
ja, bei admin dann NONE.

Reply:
ok sollte so jetzt passen